### PR TITLE
Update Alegreya to Adobe late-2013 files

### LINKED ISSUE #18: Alegreya kerning missing in the generated TTF
I'm migrating this from [Google Code issue 295](https://code.google.com/p/googlefontdirectory/issues/detail?id=295) as there was no resolution, and this impacts anyone who chooses to use Alegreya.

> The kerning seems to be missing in the generated font files.
> 
> For example, W and e ("We") in Alegreya-Regular should have -75 kerning between them according to Alegreya-Regular-TTF.vfb. According to Fontforge, this does not seem to be the case in the generated TTF-file. Similar issues are present also in the other weights and styles.

### PR DESCRIPTION
**Pull request by adrientetar**
Closes #18.

r? @davelab6

### CODE REVIEW

**Comment by adrientetar** [comment]
I found out that italicAngle wasn't set up correctly in italic files so I round-tripped through ttx and corrected it to -7.0.  
Also it seems that overlaps weren't removed in the AlegreyaSC files provided by Adobe, except for the regular weight which was manually hinted so I left the SC variant for a followup PR.

cc @gregveen @juandelperal

File: ofl/alegreya/METADATA.json [hunk 1]
```diff
@@ -8,63 +8,61 @@
...
+      "weight": 700,
+      "filename": "Alegreya-Bold.ttf",
+      "copyright": "Copyright (c) 2011, Juan Pablo del Peral (juan@huertatipografica.com.ar), with Reserved Font Names \"Alegreya\" \"Alegreya SC\". Copyright (c) 2013 by Adobe Systems Incorporated. All Rights Reserved."
     }
   ],
   "subsets": [
-    "latin",
-    "latin-ext",
-    "menu"
+    "latin"
```
Comment (adrientetar) [inline ofl/alegreya/METADATA.json]: @davelab6 The removal of latin-ext and menu seems spurious?

Comment (adrientetar) [inline ofl/alegreya/METADATA.json]: Also it seems weird that the size field at beginning of the file did not change.

**Comment by adrientetar** [comment]
@davelab6

**Comment by adrientetar** [comment]
I signed a CLA.

**Comment by juandelperal** [comment]
Thanks @adrientetar

**Comment by SverreStausland** [comment]
@adrientetar Are kerning pairs also missing from the AlegreyaSC fonts, and if so, do you also plan to add the missing kerning pairs to these fonts soon? And thanks for fixing the Alegreya font files!

**Comment by adrientetar** [comment]
@SverreStausland The AlegreyaSC fonts I got have overlaps so I’d need to remove overlap and regenerate from the fontlab source but I don’t really use fontlab so I would have to make sure that the fonts are production-ready – I haven’t spent the time doing this so far.

**Comment by juandelperal** [comment]
Hi @adrientetar @SverreStausland , I have the original files of the family. Obviously I don't have write access in this repo, but I can make a PR.

**Comment by SverreStausland** [comment]
With the original font designer @juandelperal helping out here, I'm sure we can get this fixed very soon! :) (Alegreya is a fabulous font, @juandelperal, and what a rare treat that you have made a significant subset of it freely available to the community!)

@adrientetar I incidentally notice by comparing your fixed Alegreya fonts with the ones available through Google that yours are only about half the size, yet have one more character: the soft hyphen U+00AD.

**Comment by adrientetar** [comment]
I suppose that since some weights are manually hinted the hints might take less space – not sure though.

**Comment by juandelperal** [comment]
He everyone. Please take a look at https://github.com/huertatipografica/Alegreya-libre
There is a mix of my last files and the ones improved by Adobe.

**Comment by adrientetar** [comment]
Cool, thanks.

**Comment by davelab6** [comment]
@juandelperal will the next update also use the mix of adobe and your files?

**Comment by davelab6** [comment]
Superceded by https://github.com/google/fonts/pull/1267

**Pull request closed without merging**